src/interfaces/Properties.ts

```diff
@@ -1,7 +1,7 @@
 import type { ReactNode } from 'react';
 
 /** Component Property types */
-export type StyleVariant = 'default' | 'secondary';
+export type StyleVariant = 'default' | 'secondary' | 'area';
```

Comment:
We **don't** want to add `area` property to all components who use `StyleVariant`, like I said in the Linear... we want it to *only* affect the `Button` component.